src/main/java/org/fisco/bcos/channel/client/Service.java

```diff
@@ -1094,6 +1094,24 @@ public void onReceiveChannelMessage2(ChannelHandlerContext ctx, ChannelMessage2
         }
     }
 
+    private void sendResponse2Node(ChannelHandlerContext ctx, TopicVerifyMessage message) {
+        try {
+            Message response = new Message();
+            response.setSeq(message.getSeq());
+            response.setResult(0);
+            response.setType((short) 0x37);
```

Comment:
Plz define a constant or enum for 0x37

Reply:
Channel protocol type already defined in enum ChannelMessageType.

Reply:
already fixed